x-pack/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/delete/confirm_delete_endpoint/index.tsx

```diff
@@ -27,8 +71,46 @@ export const ConfirmDeleteEndpointModal: React.FC<ConfirmDeleteEndpointModalProp
       onCancel={onCancel}
       onConfirm={onConfirm}
       title={i18n.DELETE_TITLE}
+      confirmButtonDisabled={deleteDisabled}
     >
-      {i18n.CONFIRM_DELETE_WARNING}
+      <EuiFlexGroup gutterSize="l" direction="column">
+        <EuiFlexItem grow={false}>{i18n.CONFIRM_DELETE_WARNING}</EuiFlexItem>
+        <EuiFlexItem grow={false}>
+          <EuiText
+            size="xs"
+            css={css`
+              font-family: ${euiThemeVars.euiCodeFontFamily};
+              font-weight: ${euiThemeVars.euiCodeFontWeightBold};
+            `}
+          >
+            {inferenceEndpoint.endpoint}
+          </EuiText>
+        </EuiFlexItem>
+        <EuiFlexItem>
+          {isFetching ? (
+            <EuiButtonEmpty
+              data-test-subj="scanningUsageText"
+              size="xs"
+              onClick={() => {}}
+              isLoading
+            >
+              {i18n.SCANNING_USAGE_LABEL}&hellip;
+            </EuiButtonEmpty>
+          ) : listOfUsages.length === 0 ? (
+            <RenderMessageWithIcon
+              icon="checkInCircleFilled"
+              color="success"
+              label={i18n.NO_USAGE_FOUND_LABEL}
+            />
+          ) : (
+            <ScanUsageResults
+              list={listOfUsages}
+              ignoreWarningCheckbox={ignoreWarningCheckbox}
+              onCheckboxChange={onCheckboxChange}
```

Comment:
nitpick - nonblocking: it could be better to name this similarly to ignoreWarningCheckbox, instead of a generic name. Considering boolean states are can be different.